packages/compass-editor/src/codemirror/query-history-autocompleter.ts

```diff
@@ -109,6 +140,13 @@ function createInfo(query: SavedQuery): {
   };
 }
 
+export type SavedQuery = {
+  lastExecuted: Date;
+  queryProperties: {
+    [propertyName: string]: any;
+  };
+};
```

Comment:
[nit] Can we move this type definition to higher up in the file where it used to be? Since the functions currently above this are using it.